01-Authorization/auth0authorization/views.py

```diff
@@ -31,15 +27,7 @@ def require_scope(f):
         @wraps(f)
         def decorated(*args, **kwargs):
             token = get_token_auth_header(args[0])
-            AUTH0_DOMAIN = os.environ.get('AUTH0_DOMAIN')
-            API_IDENTIFIER = os.environ.get('API_IDENTIFIER')
-            jsonurl = req.urlopen('https://' + AUTH0_DOMAIN + '/.well-known/jwks.json')
-            jwks = json.loads(jsonurl.read())
-            cert = '-----BEGIN CERTIFICATE-----\n' + jwks['keys'][0]['x5c'][0] + '\n-----END CERTIFICATE-----'
-            certificate = load_pem_x509_certificate(cert.encode('utf-8'), default_backend())
-            public_key = certificate.public_key()
-            decoded = jwt.decode(token, public_key, audience=API_IDENTIFIER, algorithms=['RS256'])
-
+            decoded = jwt.decode(token, verify=False)
```

Comment:
sounds like `verify=false` should not be here. What is the reason to remove it @alexisluque ?

Reply:
@lbalmaceda I don't know if I properly understood the question.
If the question is if `verify` is a valid parameter of `decode`, then the answer is [yes](https://pyjwt.readthedocs.io/en/latest/usage.html#reading-the-claimset-without-validation).
If the question is why are we skipping token verification, is because at this point the token was previously verified by `drf-jwt`. So, we can decode the token without verification and avoid the overhead of fetching the public key.